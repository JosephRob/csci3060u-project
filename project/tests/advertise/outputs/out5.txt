Enter command 'login' to login, or 'cancel' to cancel
> Good bye
