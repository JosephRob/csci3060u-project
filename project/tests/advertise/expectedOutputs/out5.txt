Enter command 'login' to login, or 'cancel' to cancel
> Username: 
Welcome to the auction, admin!
Type: AA	Balance: $1e+06

Menus: 
add = Add balance
advertise = Advertise an item
bid = Bid on an item
create = Create new user
delete = Delete a user
refund = Refund an item

list = List all items available

logout = Log out

What would you like to do? (Type 'menu' to display menu) 
=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+ADVERTISE+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+
Item Name (e.g. Macbook Pro, max 25 character including spaces): 
Item name should be between 1 to 25 characters
Enter item name again or cancel to cancel
Item name: 
Starting Price (e.g. 15.31 for $15.31, max $999.99): $
Not a valid value
Please enter valid value, or cancel to cancel
Starting Price (e.g. 15.31 for $15.31, max $999.99): $
Not a valid value
Please enter valid value, or cancel to cancel
Starting Price (e.g. 15.31 for $15.31, max $999.99): $
Days (e.g. 7 for 7 days, max 999 days): 
Not a valid value
Please enter valid value, or cancel to cancel
Days (e.g. 7 for 7 days, max 999 days): 
Advertise process is done
=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=END=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+=+

What would you like to do? (Type 'menu' to display menu) Good bye
